android/libraries/rib-base/src/main/java/com/badoo/ribs/core/view/ViewFactory.kt

```diff
@@ -2,4 +2,8 @@ package com.badoo.ribs.core.view
 
 import android.view.ViewGroup
 
+// TODO Replace ViewFactory with this
+interface ViewFactory2<D, T>: (D) -> (ViewGroup) -> T
```

Comment:
Also, update FooBar template for plugin?

Reply:
Templates, tutorials, and other RIBs were not touched. I'll do it if approach in general is approved.